pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -2528,6 +2528,9 @@ type RestoreSpec struct {
 	// Additional volume mounts of component pod.
 	// +optional
 	AdditionalVolumeMounts []corev1.VolumeMount `json:"additionalVolumeMounts,omitempty"`
+	// TolerateSingleTiKVOutage indicates whether to tolerate a single failure of a store without data loss
+	// +kubebuilder:default=false
+	TolerateSingleTiKVOutage bool `json:"tolerateSingleTiKVOutageOutage,omitempty"`
```

Comment:
Suggest to modify it to int to indicated how many failure tikv instances we can tolerate. It is more flexible and can help to write conditions more easily.